src/common/BrowseBack.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
```

Comment:
schoene komponente! :) vielleicht koenntest du es in BrowseBackArrow umbenennen, damit im code klar ist, dass es sich um einen Arrow, bzw. button oder sowas handelt